app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala

```diff
@@ -22,7 +22,14 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val cookieBannerAssetsPrefix: String = config.getOptional[String]("cookie-banner.assets-prefix").getOrElse("")
+  private def loadConfig(key: String): String =
+    config
+      .getOptional[String](key)
+      .getOrElse(throw new Exception(s"Missing configuration key: $key"))
+
   val footerLinkItems: Seq[String] = config.getOptional[Seq[String]]("footerLinkItems").getOrElse(Seq())
+  val trackingConsentPath: String = loadConfig("microservice.services.tracking-consent-frontend.path")
+  val trackingConsentBaseUrl: String = servicesConfig.baseUrl("tracking-consent-frontend")
```

Comment:
If you use servicesConfig.getString, you can remove the loadConfig function.

Reply:
Perfect, that's what I was looking for.